src/main/kotlin/com/re/back/dtos/request/auth/RegisterRequestDto.kt

```diff
@@ -0,0 +1,30 @@
+package com.re.back.dtos.request.auth
+
+import com.re.back.entities.auth.AppUser
+import jakarta.validation.constraints.*
+
+data class RegisterRequestDto(
+    @field:Email(
+        message = "Invalid email format. Please provide a valid email address."
+    )
+    @field:NotNull
+    @field:NotBlank
+    @field:NotEmpty
```

Comment:
We can only have notBlank validation here as it checks if the string is not null or empty